.gitignore

```diff
@@ -5,9 +5,6 @@ erl_crash.dump
 /_build
 /doc/*
 !/doc/overview.edoc
-!/doc/README.md
-!/doc/github-markdown.css
-!/doc/prism.css
-!/doc/prism.js
 /mix.lock
 /nonode@nohost/
+rebar3
```

Comment:
Could you please remove this change? It is unrelated to the pull request.

Reply:
Done